src/window.rs

```diff
@@ -25,6 +25,29 @@ use MouseCursor;
 use cli::Options;
 use config::WindowConfig;
 
+/// Default text for the window's title bar, if not overriden.
+///
+/// In X11, this the default value for the `WM_NAME` property. Other
+/// platforms haven't been documented yet.
+///
+/// ```ignore
+/// $ xprop | grep WM_NAME
+/// WM_NAME(STRING) = "Alacritty"
+/// ```
+pub const DEFAULT_TITLE: &str = "Alacritty";
```

Comment:
I think the `DEFAULT_CLASS` documentation is good the way it is. It's true that it is only relevant on X11 and it makes sense to document that there's two different window classes for anyone who isn't familiar with the way X11 works here.

However the only reason why I haven't merged this yet is because I feel like the documentation on the `DEFAULT_TITLE` still aren't optimal. And I wouldn't want to add any misleading documentation.

One issue I have with it is the codeblock, this only works on X11 and doesn't really provide a lot of additional information. I'd remove it but it should at least be mentioned that this is only x11 relevant.

The other issue is the `Other platforms haven't been documented yet.` part. If the decision is made to document them, I feel like that should be done now rather than putting a comment in which is basically a `TODO` which probably will never be fixed :P. The other alternative is that this information isn't relevant. `WINDOW_CLASS` is quite the abstract concept so mentioning the property it's attached to makes sense. However for the window title it should be fine to just say it's the default text for the window's title bar and everyone should know what's going on.

I'm probably being way too picky about this, but I thought I'd rather voice my thoughts than letting this sit forever without doing anything, so the issue can be resolved and this can be merged as quickly as possible. The actual important stuff of this PR looks great, thanks for your work @nixpulvis :+1:.

Reply:
You're not being too picky, these are all valid concerns. I've fixed the issues, and pushed an update. Let me know if there's anything else that needs work.